gpiocontrol/source/android/app/src/main/java/org/kaaproject/kaa/examples/gpiocontol/model/Device.java

```diff
@@ -42,22 +42,30 @@ public Device(String model, String deviceName, List<GpioStatus> gpioStatusList,
         this.kaaEndpointId = kaaEndpointId;
     }
 
-    public String getModel() {
-        return model;
+    @Override
```

Comment:
In This class please move methos to some order - public methods, Override, private